packages/docs/content/foundation/typography/headings.mdx

```diff
@@ -163,6 +163,25 @@ This is an optional header design.
 
 ## Code
 
+<Alert variation="warn" heading="First and last children">
+
+  The design system heading styles have alternate margins for headings that are the first or last child of their parent element. Headings that are the first child have no top margin, and headings that are last children have no bottom margin. Some examples of what we mean:
+
+  ```html
+    <form>
```

Comment:
this is nit-picky, but why is this example inside a <form> when there are no other form elements in the example?

Reply:
Yeah, maybe that's weird. I was going off of the example in a conversation I had with a designer, but I can change it to be more generic. What I was trying to do was show that this thing might be the first thing _in your form_, but it doesn't make it the first child. That can apply to a plain div too or even `<main>`